index.html

```diff
@@ -0,0 +1,490 @@
+                                        </div>
+                                        <div class="col-8 d-flex justify-content-start">
+                                            <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle mt-2" style="width: 25px; height: 25px;">
+                                            <div class="ms-2 comment-text">
+                                                <span style="font-size: 10px;" class="">You asked me for a simple strategy, so here it is. I ...</span>
+                                            </div>
+                                        </div>
+                                    </div>
+                                </div>
+                            </div>
+                        </div>
+                    </section>
+                    <section>
+                        <div class="comment-section">
+                            <div class="comment-no-and-sorts d-flex justify-content-start">
+                                <div class="me-5">1,123 Comments</div>
+                                <div class="d-flex justify-content-start">
+                                    <i data-feather="align-left"></i>
+                                    <p class="ms-2">SORT BY</p>
+                                </div>
+                            </div>
+                            <div class="form-section d-flex justify-content-start">
+                                <div class="commenter-image col-1">
+                                    <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
+                                </div>
+                                <div class="form-box col-11">
+                                    <div class="form-group"></div>
+                                    <form action="#" method="post">
+                                        <textarea class="comment-msg form-control" name="comment-msg"  id="exampleFormControlTextarea1" rows="1" placeholder="Add a comment..."></textarea>
+                                        <div class="d-flex justify-content-end mt-2">
+                                            <button type="button" class="btn btn-dark cancel-button">CANCEL</button>
+                                            <button type="button" class="btn btn-primary ms-3">COMMENT</button>
+                                        </div>
+                                    </form>
+                                </div>
+                            </div>
+                            <div class="users-comments">
+                                <div class="ind-user-comment d-flex justify-content-start">
+                                    <div class="commenter-image col-1">
+                                        <img src="img/trey-songz.jpg" alt="Trey Songz" class="rounded-circle m-auto" style="width: 40px; height: 40px;">
```

Comment:
Please do not use inline styles. Keep your style definitions in the external CSS file you created.